sdk-base/src/main/java/com/mapbox/maps/plugin/annotation/Annotation.kt

```diff
@@ -87,5 +92,10 @@ abstract class Annotation<T : Geometry>(
      * Minimum latitude value in Mercator projection.
      */
     const val MIN_MERCATOR_LATITUDE = -85.05112877980659
+
+    /**
+     * Unique feature identifier incremental counter.
+     */
+    private var FEATURE_IDENTIFIER = 5234
```

Comment:
should it be in `companion` block? I guess you want to get unique id for each instance of objects

Reply:
code in the activity is not ideal for now, I'll revisit it and introduce smaller functions a bit later.

Reply:
revisited